frontend/ui/widget/inputtext.lua

```diff
@@ -707,6 +707,26 @@ function InputText:searchString(str, case_sensitive, start_pos)
     return char_pos
 end
 
+--- Return the character at the given offset. If is_absolute is truthy then the
+-- offset is the absolute position, otherwise the offset is added to the current
+-- cursor position (negative offsets are allowed).
+function InputText:getChar(offset, is_absolute)
+    local idx
+    if is_absolute then
+        idx = offset
+    else
+        idx = self.charpos + offset
+    end
+    if idx < 1 or idx > #self.charlist then return end
+    return self.charlist[idx]
+end
+
+--- Return the current character position and the currently displayed top line
+-- index.
+function InputText:getCharPos()
```

Comment:
Is this used ?
Nice to think about the future, but if I ever want something like that, I may want to get other info than top_line_num, and I would need to look around who are the current users, and I wouldn't find any, and I would be confused :)

Reply:
No it's not used since `getChar` supports relative positions. I'll remove it.